api/data_refinery_api/urls.py

```diff
@@ -102,6 +104,8 @@ class AccessUser:
 
         # Dashboard Driver
         url(r'^stats/$', Stats.as_view(), name='stats'),
+        url(r'^stats/failed-downloader$', FailedDownloaderJobStats.as_view(), name='stats_failed_downloader'),
+        url(r'^stats/failed-processor$', FailedProcessorJobStats.as_view(), name='stats_failed_processor'),
```

Comment:
What about adding this to `stats/failures/[type]`

Reply:
I like this! If we were to go this route, do we need to have something on `/failures`?